src/Interface/InterfaceView/InterfaceView.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { KeyValue, Col, Row } from '@folio/stripes-components';
+
+import BoolToCheckbox from '../../Utils/BoolToCheckbox';
```

Comment:
please don't use this component (or whatever this is) in the new code. It's over-engineering and was introduced to reduce duplication mark on sonarcloud.

Reply:
Moreover, there is at least one weird property, like `name`. Could you please describe the purpose of it? Maybe I'm missing something

Reply:
Fixed. According to the prop name, we need to provide this property of Interface object, this field is on the provided mock-ups

Reply:
It's an view, not form . I don't see the prop `name` is used anywhere